plugins/cassandra/cassandrawu.cpp

```diff
@@ -1158,7 +1158,7 @@ static const CassandraXmlMapping wuGraphRunningMappings [] =
     {"partition", "int", NULL, hashRootNameColumnMapper},
     {"wuid", "text", NULL, rootNameColumnMapper},
     {"graphID", "text", NULL, stringColumnMapper},
-    {"subgraphID", "int", NULL, intColumnMapper},
+    {"subgraphID", "bigint", NULL, bigintColumnMapper},
```

Comment:
I thought activity/node id's and subgraph id's were actually restricted to 32bit..